modules/metering-install-configure-storage.adoc

```diff
@@ -0,0 +1,215 @@
+        createBucket: true
+----
+
+The secretName should be the name of a secret in the Metering namespace containing the AWS credentials in the data.aws-access-key-id and data.aws-secret-access-key fields.
+
+Below is an example secret file:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: your-aws-secret
+data:
+  aws-access-key-id: "MY_AWS_ACCESSS_KEY_ID"
+  aws-secret-access-key: "MY_AWS_SECRET_ACCESS_KEY"
+----
+
+The aws-access-key-id and aws-secret-access-key credentials
+must have read and write access to the bucket. For an example of an IAM policy
+granting the required permissions, see the aws/read-write.json file below.
+
+[source,yaml]
+----
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "1",
+            "Effect": "Allow",
+            "Action": [
+                "s3:AbortMultipartUpload",
+                "s3:DeleteObject",
+                "s3:GetObject",
+                "s3:HeadBucket",
+                "s3:ListBucket",
+                "s3:ListMultipartUploadParts",
+                "s3:PutObject"
+            ],
+            "Resource": [
+                "arn:aws:s3:::operator-metering-data/*",
+                "arn:aws:s3:::operator-metering-data"
+            ]
+        }
+    ]
+}
+----
+
+If you want the metering operator to create a new bucket, then see the aws/read-write-create.json example below, which contains permissions for creating and deleting buckets.
+
+[source,yaml]
+----
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "1",
+            "Effect": "Allow",
+            "Action": [
+                "s3:AbortMultipartUpload",
+                "s3:DeleteObject",
+                "s3:GetObject",
+                "s3:HeadBucket",
+                "s3:ListBucket",
+                "s3:CreateBucket",
+                "s3:DeleteBucket",
+                "s3:ListMultipartUploadParts",
+                "s3:PutObject"
+            ],
+            "Resource": [
+                "arn:aws:s3:::operator-metering-data/*",
+                "arn:aws:s3:::operator-metering-data"
+            ]
+        }
+    ]
+}
+----
+
+[id="metering-storing-data-in-microsoft-azure_{context}"] 
+== Storing data in Microsoft Azure 
+
+To use Azure blob storage you must use an existing container. Edit the default metering configuration file to specify the container name and your Azure secret. You can optionally specify a directory in the container where you want to store your data.
+
+Below is an example configuration file:
+
+[source,yaml]
+----
+apiVersion: metering.openshift.io/v1
+kind: MeteringConfig
+metadata:
+  name: "operator-metering"
+spec:
+  storage:
+    type: "hive"
+    hive:
+      type: "azure"
+      azure:
+        container: "bucket1"
+        secretName: "my-azure-secret"
+        rootDirectory: "/testDir"
+----
+
+Your secret should be in the metering namespace and use the following format:
```

Comment:
Is the namespace actually called "metering" or is it something like `openshift-metering`? I'd get specific if possible. Elsewhere throughout the PR.